### ISSUE TITLE
Add ability to temporary disable some upload host from the list

### ISSUE THREAD

**Issue by zhartus** [ribtoks/xpiks#648]
It can be implemented as checkbox "Temporarily disabled" in "Advanced" section. And as a result, such a host cannot be checked for upload, and can also marked with a pale color.

Here is an example of using the case: Crestock currently does not accept images for publication, as they are preparing some major changes in the infrastructure. It is important to have such information before your eyes, so as not to try to load a stock that is not working at the moment.

By the way, it would also be nice to have a user comment field for each upload host, but this is not so important.

**Comment by ribtoks** [comment]
In they way you described it this is questionable since there should be no way to disable something on users computers. What if they want to upload even if it's "prohibited"? The only nice way to solve that is to display "information" icon next to a stock if there's something wrong with it. But this needs to be properly thought through and discussed.

**Comment by zhartus** [comment]
> What if they want to upload even if it's "prohibited"?

Well, in this case they will able to uncheck "Temporarily disabled" checkbox at any time. What I offer is just a little protection from unwanted uploads.

**Comment by ribtoks** [comment]
I understand that you want some protection from uploading to stocks at specific times, however your current proposed solution does not seem to be very user friendly to me (as a user how would I know about existence of some special checkbox that disables my upload). For now I will park this discussion, but I will keep this issue opened and I will think about this problem later. Thanks for raising this!

**Comment by zhartus** [comment]
> your current proposed solution does not seem to be very user friendly to me

Then your own idea can be a good solution:

> The only nice way to solve that is to display "information" icon next to a stock if there's something wrong with it.

I can offer a more detailed scenario to this: on the “Advanced” tab add the checkbox “Add comment”, and after checking it, allow the user to enter a comment, which will be displayed when you hover over the «I»-icon. Naturally, the icon should appear next to the host name only if there is a comment.

P.S.: Якщо я не помиляюся, ви з України? Я також)
(пізніше видалю цю частину комменту)

**Comment by ribtoks** [comment]
Comment-based system is not a solution for these kind of problems. Comments need to be updated and users are lazy. Also the main purpose of such system is to notify in advance, not after that happened. 

Also if such system will be used by less than 1% of users (however currently nobody except you asked for it) it is not worth the effort to implement. No offense intended, but remember, my work is free for you.

P.S. Так, я родом з України.

**Comment by zhartus** [comment]
> it is not worth the effort to implement. No offense intended, but remember, my work is free for you.

Of course. All proposals on my part do not commit to anything. All final decisions are made only by you, I fully understand this. And thank you very much for the wonderful work you are doing.
